src/components/ChatBotInput/ChatBotInput.tsx

```diff
@@ -187,13 +205,15 @@ const ChatBotInput = ({
 				onBlur={handleBlur}
 			/>
 			<div className="rcb-chat-input-button-container">
-				{!botOptions.voice?.disabled && isDesktop &&
-					<VoiceButton inputRef={inputRef} textAreaDisabled={textAreaDisabled}
-						voiceToggledOn={voiceToggledOn} handleToggleVoice={handleToggleVoice}
-						triggerSendVoiceInput={triggerSendVoiceInput} setInputLength={setInputLength}
-					/>
-				}
-				<SendButton handleSubmit={handleSubmit}/>
+				{botOptions.chatInput?.buttons?.map((button, index) => {
+					if (button === "send"){
```

Comment:
Hey! The whitespace is still missing between brackets `){`. Same for the `else-if` blocks below.